### DISCUSSION TITLE
Keep back monitor on when HDMI connected.

### DISCUSSION THREAD

**Discussion by DoubleACS** [gopro/labs#983] [Feature Requests]
I am not sure if this is already a feature but it sure would be helpful if possible.  We produce a lot of baseball and softball and we use a gopro for a backstop camera.  The camera is attached to the fence.  We connect an HDMI cable to the media mod to go back to our production van.  During a game on occasion the ball with hit the fence and shift the camera ever so slightly.  We would like to be able to adjust the camera without having to disconnect the HDMI cable.

**Comment by dnewman-gpsw** [comment]
How do you see camera firmware helping?  This would be more solvable with software in the production van to reframe the video feed.

**Reply by DoubleACS** [reply]
Thank you for responding.  All of our productions are for television so as soon as you start cropping a 1080 feed the image starts to breakdown very quickly.  Especially, depending on how much the camera has shifted.  I see the firmware helping by allowing our PA to adjust the camera without having to disconnect the HDMI cable.  The PA would be able to see their adjustments.  I would think this could only be accomplished with a firmware adjustment.

**Reply by dnewman-gpsw** [reply]
Sorry I didn't understand.  Moving the camera is physical, the firmware is not.  How can firmware help move the camera?

**Reply by DoubleACS** [reply]
The firmware doesn't move the camera.  

If I have an HDMI cable plugged into the media mod, can I see the monitor?  NO.  A firmware update/option to keep the monitor ON when an HDMI cable is plugged IN is what I am requesting.

**Reply by dnewman-gpsw** [reply]
I still don't understand.  Why are you disconnecting the HDMI? Wouldn't the HDMI continue to provide a signal while you move/adjust the camera?  If disconnected, HDMI can't continue to be live, as HDMI is a negotiation between the receiver and transmitter.

**Reply by DoubleACS** [reply]
Yes, the HDMI continues to provide a signal but the camera is located 100-200 feet (HDMI converted to SDI) away from the production van.  So when the PA goes to adjust the camera they have no choice but to disconnect the HDMI cable to see the camera view.  

If there was a firmware option to keep the gopro monitor on while the HDMI cable is plugged in, problem solved.

**Reply by dnewman-gpsw** [reply]
Okay, I understand. That is not possible, the internal hardware either drives the LCD or the HDMI.  You will need to use the Quik app for the feed monitoring as that shouldn't be affected by the HDMI connection.

**Reply by DoubleACS** [reply]
ok, so that is a hardware limitation not a software/firmware limitation.  Thank you.
